api/src/s2-api.ts

```diff
@@ -90,7 +90,62 @@ async function getPaper(s2Id: string, apiKey?: string): Promise<Paper | undefine
   );
   const response = await axios.get<S2ApiPaper>(`${apiOrigin}/paper/${s2Id}`, conf);
```

Comment:
Could this change to `getPaperUncached()`? That way you don't have to duplicate as much code?

Reply:
Yeah, I think there's a way to collapse the two now that you mention it.